src/app/project-page/project-page.component.html

```diff
@@ -125,15 +125,18 @@ <h1 class="flex-grow-1">{{project.name}}</h1>
       <tbody>
         <tr routerLink="/project/{{project.id}}/issue/{{issue.id}}" *ngFor="let issue of getIssuesForCurrentPage()">
           <td scope="row">
-            <span class="issue-summary">{{issue.summary}}</span>
+            <span class="issue-summary">
+              {{issue.summary && issue.summary.length > 75 ? (issue.summary | slice:0:72) + '...' : issue.summary}}
```

Comment:
I know there's a .text-truncate class that's supposed to do this automatically but I couldn't figure it out either.